sceptre/cli/update.py

```diff
@@ -57,23 +57,38 @@ def update_command(ctx, path, change_set, verbose, yes):
         try:
             # Wait for change set to be created
             statuses = plan.wait_for_cs_completion(change_set_name)
-            # Exit if change set fails to create
+
+            at_least_one_ready = False
+
             for status in list(statuses.values()):
```

Comment:
No need to cast the values to a list if you're iterating over them. 
```suggestion
            for status in statuses.values():
```

Reply:
Done.